### ISSUE TITLE
Compose window: option to switch between reply and reply all?

### ISSUE THREAD

**Issue by orschiro** [nylas/N1#1935]
Dear all,

What do you think of adding an option to switch between reply and reply all in composing mode?

This is useful if the user has started to compose a response but has forgotten to check whether to reply to all or not.

In this case, currently I have to withdraw the draft and re-do the composing since I cannot switch between both replying modes without losing my content.

Thankfully,

~Robert

**Comment by bengotow** [comment]
Hey! Currently there's no button for this, but you can toggle an existing draft from reply to reply-all, etc. by pressing the Reply / Reply All keyboard shortcuts while editing the draft.

**Comment by orschiro** [comment]
Hi Ben,

  

Where do I have to put the focus?

  

If I press `a` and `r` to switch between both modes, then these characters are
entered into the body of the composing window.

  

Thankfully,

  

~Robert

  
Sent from [Nylas N1](https://nylas.com/n1?ref=n1), the extensible, open source
mail client.  

On Apr 13 2016, at 7:21 pm, Ben Gotow &lt;notifications@github.com&gt; wrote:  

> Hey! Currently there's no button for this, but you can toggle an existing
draft from reply to reply-all, etc. by pressing the Reply / Reply All keyboard
shortcuts while editing the draft.

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1935#issuecomment-209553015)![](https://github.com/notifications/beac
on/AAKbwz8N4_airrubB_HO4wh_aqGHHQe0ks5p3SYqgaJpZM4IFgyE.gif)

**Comment by vince-p** [comment]
I agree that it would be very helpful to have a button to switch between reply and replyall

**Comment by orschiro** [comment]
Still very much missing this behaviour. If I reply to just one person, I would envision such a button in the top right to switch to reply-all. 

![image](https://cloud.githubusercontent.com/assets/170947/16292973/893b13a0-3916-11e6-884f-b5a697cdf0f6.png)
